src/color.rs

```diff
@@ -102,6 +104,8 @@ impl Elem {
             Elem::Exec => theme.permission.exec,
             Elem::ExecSticky => theme.permission.exec_sticky,
             Elem::NoAccess => theme.permission.no_access,
+            Elem::Acl => theme.permission.acl,
+            Elem::SecurityContext => theme.permission.security_context,
```

Comment:
What do you think about renaming it to just `context`?

Reply:
It thought that was a bit too abstract. I'm mostly used to context being whatever opaque structure user has to drag about to interact with C libraries like openssl or glib.
I'm not too hung up on it. I'll rename it if you want.

Reply:
Since this is under `theme.permission` I was thinking that `context` alone would be clear enough. Plus, the flag is just `--context`. The main idea behind this was to make the name in theme file consistent with `--context` flag.